mappings/net/minecraft/entity/mob/MobEntity.mapping

```diff
@@ -9,6 +9,9 @@ CLASS net/minecraft/class_1753 net/minecraft/entity/mob/MobEntity
 	FIELD field_7499 armorInventory [Lnet/minecraft/class_2056;
 	FIELD field_7500 pickUpLoot Z
 	FIELD field_7501 persistent Z
+	FIELD field_7502 leashes Z
```

Comment:
not a good name. this doesnt feel like a yarn name

Reply:
took it from nbt

Reply:
typo